library/bignum_core.c

```diff
@@ -434,4 +434,18 @@ void mbedtls_mpi_core_montmul( mbedtls_mpi_uint *X,
     mbedtls_ct_mpi_uint_cond_assign( AN_limbs, X, T, (unsigned char) ( carry ^ borrow ) );
 }
 
+int mbedtls_mpi_core_get_mont_R2_unsafe( mbedtls_mpi *X,
+                                         mbedtls_mpi const *N )
```

Comment:
(minor, more a comment than a request for change) we normally do `const mbedtls_mpi *N`

Reply:
Changed as requested